docs/background_info/meta_attributes.md

```diff
@@ -0,0 +1,96 @@
+> To a metaelement metaattributes could be compositional assigned (class "MetaAttribut"). The metaattributes are neither defined by the language INTERLIS nor by the meta model. They exist that the information that exceed INTERLIS could still be a component of the model data. The metaattributes has a name (attribute "Name"), that have to be unique among the the metaattributes of this metaelement, and it has a value (attribute: "Value").
+
+Translated from source: [STAN_d_DEF_2011-06-22_eCH-0117 Meta-Attribute für INTERLIS-Modelle.pdf](https://www.ech.ch/alfresco/s/ech/download?nodeid=788eb38a-bf2b-4f3d-96a8-addc37bba41f) (German) from eCH www.interlis.ch
```

Comment:
The original source of this quote is in [2] KOGIS (2008): INTERLIS 2-Metamodell.
Online http://www.interlis.ch/interlis2/docs23/ili2-metamodel_2008-08-
29_d.pdf

Im Dokument zum INTERLIS 2-Metamodell [2] steht dazu:
«Einem Metaelement können kompositorisch Metaattribute zugeordnet sein (Klasse "MetaAttribut").
Metaattribute sind weder durch die Sprache INTERLIS noch durch das Metamodell
genauer definiert. Sie dienen insbesondere dazu, dass Informationen, die über INTERLIS
hinausgehen, als Bestandteil der Modelldaten beigefügt werden können. Metaattribute
haben einen Namen (Attribut "Name"), der innerhalb der Metaattribute desselben Metaelementes
eindeutig sein muss, und den beschreibenden Wert (Attribut "Value").»